docs/fenix-engineers-root.md

```diff
@@ -1,7 +1,65 @@
 ---
 id: fenix-engineers-root
-title: Root Page
+title: Implementing an experiment
 slug: /fenix-engineers
 ---
 
-TODO: Update this page introduce the Fenix Engineers section of the docs. Or, delete it.
+## Client configuration prechecks
+✅  Get a copy of the experiment brief for your experiment from the [Experiments Repository](https://mana.mozilla.org/wiki/x/FaXNBw) mana page.
+
+✅  Assure that your experiment brief contains values for:
+  - experiment name
+  - featureID
+  - branch names (for a basic a/b experiment, prefer *control* and *treatment*)
+
+✅  Confirm the **featureID** is added to the experimenter list of console values.  
+> NOTE: featureID setup access is controlled via experimenter Django admin.
+> - **staging**, ask for help in the #nimbus-rust-sdk slack channel
+> - **production**, ask for help in the #experimenter slack channel
+
+## Fenix setup
+
+1. Update [experiments.kt](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/experiments/Experiments.kt) **Experiment** to include the featureID string as your experiment constant.
+2. Confirm [experiments.kt](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/experiments/Experiments.kt) **ExperimentBranch** contains the branch names you intend to use or add them if you are using something other than the *control* and *treatment* default values.
+3. Initialize your experiment in the appropriate location and call **getExperimentBranch** to get your experiment.
+
+### Example
+
+This is an example of the configuration of a Fenix **Bookmarks A/B** experiment on the “**Bookmarks Icon**” (bookmark-icon) featureID using the **treatment** and **control** branches.
+
+**Experiments.kt**
+
+```
+class Experiments {
```

Comment:
Fun tip: Docusaurus markdown supports a title for the syntax block, it's written as

```
```kt title="Experiments.kt"
class Experiments {
```

![Screen Shot 2021-03-16 at 4 40 44 PM](https://user-images.githubusercontent.com/6392049/111376908-6e510b00-8676-11eb-97d5-78567de3e942.png)

Reply:
sadly no syntax highlighting yet

Reply:
thank you for the tip, looks much better like that.